api/app_sph_lms/api/serializer/lesson_serializer.py

```diff
@@ -0,0 +1,8 @@
+from app_sph_lms.models import Lesson
+from rest_framework import serializers
+
+
+class LessonSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Lesson
+        fields = ('title', 'link', 'description')
```

Comment:
Please pass `id` also, its needed on FE
```suggestion
        fields = ('id', 'title', 'link', 'description')
```